README.md

```diff
@@ -274,7 +274,8 @@ TODO-C:   TBD (list services, timers, jobs)
 TODO-C:   Those have the following security features: TBD (add details like
 TODO-C:   reduced permissions, temp environment, restricted users/groups,
 TODO-C:   seccomp, apparmor, ...)
-TODO: - Packages does not open privileged ports (ports < 1024)
+TODO: - Packages does not open privileged ports (ports < 1024) or has
+TODO:   a reason to do so (TBD)
```

Comment:
This statement is a bit ambiguous: does it mean:
- this package has no reason to open privileged ports?
- this package opens privileged ports but it has a reason to do so?

Reply:
Indeed, @dvierbe already provided an improvement which I included in this PR - thank you both!